content/en/dashboards/guide/tv_mode.md

```diff
@@ -16,13 +16,13 @@ To ensure that your dashboard displays correctly on a TV, follow these steps:
 1. **Design Your Dashboard**: Begin by creating your dashboard in Datadog. Focus on arranging your widgets within the 12-column grid layout that Datadog dashboards utilize. Keep in mind that the aspect ratio of your widgets and the overall dashboard affects how they are displayed in TV mode.  
 2. **Enable TV mode**: When your dashboard is ready, enable TV mode. Do this while your screen is connected to the TV and in full-screen mode. This step ensures that the dashboard automatically adjusts to fit the TV screen without the need for manual resizing.  
     {{< img src="/dashboards/guide/tv_mode/tv_mode_config_option.png" alt="Enable the TV mode option through the dashboard Configure menu" style="width:100%;" >}} 
-3. **Optimize Display Settings**: If your dashboard content is not filling the screen edges, you can simulate a large screen by zooming in or out. Use keyboard shortcuts to adjust the browser display before re-entering TV mode, `CMD/CTRL + +(plus)` to zoom in and  CMD/CTRL + -(minus)` to zoom out.
+3. **Optimize Display Settings**: If your dashboard content is not filling the screen edges, you can simulate a large screen by zooming in or out. Use keyboard shortcuts to adjust the browser display before re-entering TV mode, `CMD/CTRL + +(plus)` to zoom in and  CMD/CTRL + -(minus)` to zoom out. **Note**: This solution comes with readability downsides; it can make some fonts smaller and difficult to read from a distance.
```

Comment:
Looks like Vale was secretly also pointing out a missing ` character:
```suggestion
3. **Optimize Display Settings**: If your dashboard content is not filling the screen edges, you can simulate a large screen by zooming in or out. Use keyboard shortcuts to adjust the browser display before re-entering TV mode, `CMD/CTRL + +(plus)` to zoom in and `CMD/CTRL + -(minus)` to zoom out. **Note**: This solution comes with readability downsides; it can make some fonts smaller and difficult to read from a distance.
```